zeebe/exporters/elasticsearch-exporter/src/test/java/io/camunda/zeebe/exporter/TestSupport.java

```diff
@@ -107,7 +107,8 @@ static Stream<ValueType> provideValueTypes() {
             ValueType.ROLE,
             ValueType.TENANT,
             ValueType.GROUP,
-            ValueType.MAPPING);
+            ValueType.MAPPING,
+            ValueType.IDENTITY_SETUP);
```

Comment:
🔧 Can you move this just under REDISTRIBUTION (if we plan to NOT export this record)